
Well, no idea why this is missing ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.11/drivers/net/sk98lin/skge.c.vanilla	2005-05-25 18:36:44.000000000 +0200
+++ linux-2.6.11/drivers/net/sk98lin/skge.c	2005-05-25 18:38:33.000000000 +0200
@@ -5152,6 +5152,8 @@
 	{ 0, }
 };
 
+MODULE_DEVICE_TABLE(pci, skge_pci_tbl);
+
 static struct pci_driver skge_driver = {
 	.name		= "skge",
 	.id_table	= skge_pci_tbl,
